portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/Form.path

```diff
@@ -31,10 +31,15 @@
 	<td></td>
 </tr>
 <tr>
-	<td>SAVE_NOTIFICATION</td>
-	<td>//div[contains(@class,'autosave-bar')]//span</td>
+	<td>FORM_TEMPORARILY_NOT_AVAILABLE</td>
+	<td>//div[contains(concat(' ',normalize-space(@class),' '),'alert')][contains(concat(' ',normalize-space(@class),' '),'alert-danger')][contains(text(),'Form is temporarily unavailable.')]</td>
 	<td></td>
 </tr>
+<tr>
+    <td>SAVE_NOTIFICATION</td>
+    <td>//div[contains(@class,'autosave-bar')]//span</td>
+    <td></td>
```

Comment:
Hi @ewertonluna, the indentation for the SAVE_NOTIFICATION is not the same as the others.